src/VisualStudio/CodeLens/ReferenceCodeLensProvider.cs

```diff
@@ -21,6 +21,8 @@
 using Microsoft.VisualStudio.Threading;
 using Microsoft.VisualStudio.Utilities;
 using Roslyn.Utilities;
+using IAsyncCodeLensDataPoint = Microsoft.VisualStudio.Language.CodeLens.Remoting.IAsyncCodeLensDataPoint;
```

Comment:
unnecessary?

Reply:
I get these errors after updating the packages .. so pretty sure something new is added, and this is needed